loom/engine/bindings/loom/lmApplicationTasks.cpp

```diff
@@ -51,7 +51,7 @@ void loom_tick()
 
     performance_tick();
 
-    profilerBlock_t p = { "loom_tick", platform_getMilliseconds(), 8 };
+    profilerBlock_t p = { "loom_tick", platform_getMilliseconds(), 17 };
```

Comment:
1000/60 = 16.666..., so 17 is that with some buffer. It's not variable like it probably should be, but it's better than 8 at least, since with 8 it just printed that it was over threshold every frame (since with 60fps vsync every frame is obviously going to be >8ms).